cluster/images/hyperkube/turnup.sh

```diff
@@ -20,13 +20,15 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+K8S_VERSION=${K8S_VERSION:-"1.1.2"}
+
 docker run --net=host -d gcr.io/google_containers/etcd:2.2.1 \
   /usr/local/bin/etcd \
     --addr=127.0.0.1:4001 \
     --bind-addr=0.0.0.0:4001 \
     --data-dir=/var/etcd/data
```

Comment:
The flags for `etcd` are kinda out-dated. Ref https://github.com/kubernetes/kubernetes/blob/master/docs/getting-started-guides/docker-multinode/master.md#startup-etcd-for-flannel-and-the-api-server-to-use
Yet you may address this in a follow-up pr:-)